src/main/java/frc/robot/commands/ScoreStageTwoRotations.java

```diff
@@ -27,7 +27,7 @@ public ScoreStageTwoRotations(ControlPanelSubsystem p) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    controlPanel.wheelUp();
+    controlPanel.setMotorSpeed(10);
```

Comment:
Same as above.  I don't think it's strictly necessary here, but I don't think it hurts either.

```suggestion
```